syntax = "proto3";

message MyData {
  string id = 1;
  string name = 2;
  int32 age = 3;
}
